### PR TITLE
all generated metafiles

### PR DESCRIPTION
**Pull request by gaurabdg**

### CODE REVIEW

File: src/main/resources/AbbreviationAsWordInNameCheck.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        </property>
+        <property name="ignoreStaticFinal" type="static" default-value="final">
+          <description><![CDATA[Allow to skip variables with both {@code static} and
+ {@code final} modifiers.]]></description>
+        </property>
+        <property name="ignoreOverriddenMethods" type="@Override" default-value="boolean">
+          <description><![CDATA[Allow to ignore methods tagged with {@code @Override}
+ annotation (that usually mean inherited name).]]></description>
+        </property>
+        <property name="tokens" type="java.lang.String[]" default-value="tokenSet">
```
Comment (romani) [inline src/main/resources/AbbreviationAsWordInNameCheck.xml:52]: > default-value="tokenSet"

real default  value should be there and tokenSet is validationType

File: src/main/resources/AbbreviationAsWordInNameCheck.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        <property name="allowedAbbreviationLength" type="int" default-value="3">
+          <description><![CDATA[Indicate the number of consecutive capital
+ letters allowed in targeted identifiers (abbreviations in the classes, interfaces, variables
+ and methods names, ... ).]]></description>
+        </property>
+        <property name="allowedAbbreviations" type="java.lang.String[]" default-value="{}">
+          <description><![CDATA[Specify list of abbreviations that must be skipped for
+ checking. Abbreviations should be separated by comma.]]></description>
+        </property>
+        <property name="ignoreFinal" type="final" default-value="boolean">
```
Comment (romani) [inline src/main/resources/AbbreviationAsWordInNameCheck.xml:38]: ???

File: src/main/resources/AbbreviationAsWordInNameCheck.xml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+          <description><![CDATA[Allow to skip variables with {@code final} modifier.]]></description>
+        </property>
+        <property name="ignoreStatic" type="static" default-value="boolean">
```
Comment (romani) [inline src/main/resources/AbbreviationAsWordInNameCheck.xml:41]: ????

File: src/main/resources/AbbreviationAsWordInNameCheck.xml (hunk 1 above)
Comment (gaurabdg) [reply src/main/resources/AbbreviationAsWordInNameCheck.xml]: resolved

File: src/main/resources/AbbreviationAsWordInNameCheck.xml (hunk 2 above)
Comment (gaurabdg) [reply src/main/resources/AbbreviationAsWordInNameCheck.xml]: resolved

File: src/main/resources/AbbreviationAsWordInNameCheck.xml (hunk 3 above)
Comment (gaurabdg) [reply src/main/resources/AbbreviationAsWordInNameCheck.xml]: resolved

**Pull request merged**